packages/ui/atomic/template/src/components/results-manager/results-manager.tsx.hbs

```diff
@@ -6,7 +6,7 @@ import template{{inc @index}} from './template-{{inc @index}}.html';
 
 /**
  * Component used to manage results & result templates.
- * See https://docs.coveo.com/en/atomic/latest/usage/create-a-result-list/
+ * See https://docs.coveo.com/en/atomic/latest/usage/displaying-results/
```

Comment:
This looks like the potential replacement link - not sure if there is a better one out there - @jpmarceau ?

Reply:
I think that's good 👍